### ISSUE TITLE
the new argument "new_command_timeout" breaks the interface

### ISSUE THREAD

**Issue by yahman72** [serhatbolsu/robotframework-appiumlibrary#26]
The new argument `new_command_timeout` in `Open Application` should have a default value of `None`. 

With the current default value `=60` it is not possible to use the appium command line option `--command-timeout` anymore. I.e. starting appium with `--command-timeout 120` has no effect, because the appium library always overrides the that 120 seconds with 60 seconds.
(and the default on appium side is anyway already 60secs i.e. starting appium without `--command-timeout`)

In Short:
instead of `new_command_timeout=60` it should be `new_command_timeout=None` 
and send it to appium only when provided

<pre>
 if new_command_timeout:
        desired_caps['newCommandTimeout'] = new_command_timeout
</pre>
